### PR TITLE
Feature/link content type

### PR DESCRIPTION
**Pull request by tiagojsag**

### CODE REVIEW

**Comment by tiagojsag** [comment]
Build is failing because of the json type column on db, need to look into it

**Comment by tiagojsag** [comment]
build is green. @santostiago please review

File: app/views/management/site_pages/edit.html.erb [hunk 1]
```diff
@@ -1,5 +1,21 @@
 <div class="wrapper">
-  <h1><%= @title %></h1>
+  <h1>Edit <%= @site_page.content_type_humanize %></h1>
 
-  <%= render @partial, site_page: @site_page, form_for_path: [:management, @site_page] unless @partial.nil?%>
+  <% case @site_page.content_type
```
Comment (santostiago) [inline app/views/management/site_pages/edit.html.erb]: This logic would be better in a helper

File: app/views/management/site_pages/new.html.erb [hunk 2]
```diff
@@ -1,8 +1,27 @@
 <div class="wrapper">
-  <h1><%= @title %></h1>
+  <%
+    case @site_page.content_type
```
Comment (santostiago) [inline app/views/management/site_pages/new.html.erb]: Same comment as management/site_pages/edit.html.erb

File: app/views/management/site_pages/show.html.erb [hunk 3]
```diff
@@ -1,33 +1,29 @@
 <div class="wrapper">
-  <p>
-    <strong>Name:</strong>
-    <%= @site_page.name %>
-  </p>
+  <%
+    case @site_page.content_type
```
Comment (santostiago) [inline app/views/management/site_pages/show.html.erb]: Same as management/site_pages/edit.html.erb

**Pull request merged**